src/framework/ui/toggle/toggle.component.tsx

```diff
@@ -213,6 +214,18 @@ export class ToggleComponent extends React.Component<ToggleProps> implements Pan
     }
   };
 
+  private getIconColor = (interpolatedColor: string | Animated.AnimatedDiffClamp)
+    : string | Animated.AnimatedDiffClamp => {
+    // this is due to react-native bug fix the same content-container rgba colors
```

Comment:
Does it also affect on `Svg` elements or on `tintColor` when using `Image`?

Reply:
I think it's better to declare a service then. Because there could be a case when using transparent colors in not only `checked` / `disabled` states (e.g custom mapping). We can check for `color.startsWith('rgba')` for example.

Reply:
Didn't get your idea. Could u please explain in a little bit more details?